dask/dataframe/io/parquet/core.py

```diff
@@ -0,0 +1,452 @@
+
+
+def read_parquet_part(func, fs, meta, part, columns, index, kwargs):
+    """ Read a part of a parquet dataset """
+    df = func(fs, part, columns, **kwargs)
+    if not len(df):
+        df = meta
+    if index:
+        df = df.set_index(index)
+    return df
+
+
+def to_parquet(
+    df,
+    path,
+    engine="auto",
+    compression="default",
+    write_index=True,
+    append=False,
+    partition_on=None,
+    storage_options=None,
+    compute=True,
+    **kwargs
+):
+    """Store Dask.dataframe to Parquet files
+
+    Notes
+    -----
+    Each partition will be written to a separate file.
+
+    Parameters
+    ----------
+    df : dask.dataframe.DataFrame
+    path : string
+        Destination directory for data.  Prepend with protocol like ``s3://``
+        or ``hdfs://`` for remote data.
+    engine : {'auto', 'fastparquet', 'pyarrow'}, default 'auto'
+        Parquet library to use. If only one library is installed, it will use
+        that one; if both, it will use 'fastparquet'.
+    compression : string or dict, optional
+        Either a string like ``"snappy"`` or a dictionary mapping column names
+        to compressors like ``{"name": "gzip", "values": "snappy"}``. The
+        default is ``"default"``, which uses the default compression for
+        whichever engine is selected.
+    write_index : boolean, optional
+        Whether or not to write the index. Defaults to True *if* divisions are
+        known.
+    append : bool, optional
+        If False (default), construct data-set from scratch. If True, add new
+        row-group(s) to an existing data-set. In the latter case, the data-set
+        must exist, and the schema must match the input data.
+    partition_on : list, optional
+        Construct directory-based partitioning by splitting on these fields'
+        values. Each dask partition will result in one or more datafiles,
+        there will be no global groupby.
+    storage_options : dict, optional
+        Key/value pairs to be passed on to the file-system backend, if any.
+    compute : bool, optional
+        If True (default) then the result is computed immediately. If False
+        then a ``dask.delayed`` object is returned for future computation.
+    **kwargs
+        Extra options to be passed on to the specific backend.
+
+    Examples
+    --------
+    >>> df = dd.read_csv(...)  # doctest: +SKIP
+    >>> to_parquet('/path/to/output/', df, compression='snappy')  # doctest: +SKIP
+
+    See Also
+    --------
+    read_parquet: Read parquet data to dask.dataframe
+    """
+    partition_on = partition_on or []
+    if isinstance(partition_on, string_types):
+        partition_on = [partition_on]
+
+    if set(partition_on) - set(df.columns):
+        raise ValueError("Partitioning on non-existent column")
+
+    if compression != "default":
+        kwargs["compression"] = compression
+    elif "snappy" in compress:
+        kwargs["compression"] = "snappy"
```

Comment:
else no compression?

Reply:
I suspect that this is a holdover from before.  My guess is that this just doesn't provide any instruction, and leaves it to the parquet library to decide.  Happy either way though.  I don't have an opinion here.